pkg/datasources/secrets.go

```diff
@@ -104,15 +104,15 @@ func Secrets() *schema.Resource {
 
 func ReadSecrets(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
 	client := meta.(*provider.Context).Client
-	req := sdk.NewShowSecretRequest()
+	req := *sdk.NewShowSecretRequest()
```

Comment:
why this change?

Reply:
ig it is not necessary so i will leave it as it was before so this is oversight from my side